recipes/thrift-cpp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: thrift-cpp
+  version: "0.9.3"
+
+source:
+  fn: thrift-0.9.3.tar.gz
+  url: http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+  md5: 88d667a8ae870d5adeca8cb7d6795442
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - boost
```

Comment:
You should be able to use clang if you just need C++11. What is required by thrift?